zerver/lib/actions.py

```diff
@@ -3429,6 +3442,19 @@ def do_change_icon_source(realm: Realm, icon_source: str, log: bool=True) -> Non
                               icon_url=realm_icon_url(realm))),
                active_user_ids(realm.id))
 
+    if realm.is_organization_profile_incomplete():
+        event = dict(
+            type='panels',
+            op='show_warn'
+        )
+        send_event(realm, event, active_user_ids(realm.id))
+    else:
+        event = dict(
+            type='panels',
+            op='hide_warn'
+        )
+        send_event(realm, event, active_user_ids(realm.id))
```

Comment:
I don't think we need any of these events.  The existing `server_events_dispatch.js` code can just call a frontend-side function to check if the fields are set or not.